src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -94,17 +126,51 @@ def update_data(self):
             else:
                 continue
 
-            vlanmac = self.fdb_vlanmac(fdb)
+            vlanmac, vlanid = self.fdb_vlanmac(fdb)
             if not vlanmac:
                 mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': failed in fdb_vlanmac().".format(fdb_str))
                 continue
+            self.tp_fdb_status_map[vlanmac] = self.get_tp_fdb_status(ent, fdb)
             self.vlanmac_ifindex_map[vlanmac] = port_index
             self.vlanmac_ifindex_list.append(vlanmac)
+
+            try:
+                ent_type = ent["SAI_FDB_ENTRY_ATTR_TYPE"]
+                if ent_type == 'SAI_FDB_ENTRY_TYPE_DYNAMIC':
+                    if vlanid in self.vlan_dynamic_count_map:
+                        self.vlan_dynamic_count_map[vlanid] = self.vlan_dynamic_count_map[vlanid] + 1
+                    else:
+                        self.vlan_dynamic_count_map[vlanid] = 1
+                self.vlan_id_list.append(vlanid)
+            except:
```

Comment:
>except [](http://example.com/codeflow?start=12&length=6)

What is the intention of this except? It catches all types, which is not common.

Reply:
dictionary access error if SAI_FDB_ENTRY_ATTR_TYPE key does not exist in ent